src/Column/ColumnStatus.php

```diff
@@ -91,7 +91,7 @@ public function getColumn(): string
 	public function getCurrentOption(Row $row): ?Option
 	{
 		foreach ($this->getOptions() as $option) {
-			if ($option->getValue() === $row->getValue($this->getColumn())) {
+			if ($option->getValue() == $row->getValue($this->getColumn())) {
```

Comment:
Check for exact "empty" values would be better than ==

Reply:
I made new PR because i deleted that old fork. https://github.com/contributte/datagrid/pull/827